keras_retinanet/utils/coco_eval.py

```diff
@@ -69,7 +69,7 @@ def evaluate_coco(generator, model, threshold=0.05):
         print('{}/{}'.format(index, generator.size()), end='\r')
 
     if not len(results):
-        return
+        return [0]
```

Comment:
Why `return [0]`? Wouldn't `return []` make more sense?

Reply:
When I test with step=1 epoch=1 training from scratch.There are no results.I can't get mAP list from`coco_eval.stats`.The original `return` raise `TypeError: 'NoneType' object is not subscriptable`. So I assign `[0]` for mAP=0 for there are no results. 
 
`return []` will raise `IndexError: list index out of range` because i write  `summary_value.simple_value = mAP[0]`in `coco.py line35`.

Reply:
In the success path, does `mAP` contain all values of the summary? Maybe it's better to iterate over the `mAP` list and add them all? Is there a way to map them to the correct name of the value?